libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -405,4 +445,25 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
       month: this.month,
     };
   }
+
+  // TODO: Should this be based on new Date() (so deterministically today) or this.todayDate?
+  // TODO: Is an if else statement more readable here?
+  private getDateFromNavigableYear(navigableYear: number | Date): Date {
+    return navigableYear instanceof Date
+      ? navigableYear
+      : new Date(
+          (this.todayDate ? this.todayDate : new Date()).getFullYear() + navigableYear,
+          0,
+          1
+        );
+  }
```

Comment:
Yes, it could be simplified with a guard instead:
```suggestion
  private getDateFromNavigableYear(navigableYear: number | Date): Date {
    if (navigableYear instanceof Date) return navigableYear;
    const today = this.todayDate || new Date();
    return new Date(today.getFullYear() + navigableYear, 0, 1);
  }
```